### PR TITLE
Fixed the example for newer Versions of Rng

### PR DESCRIPTION
**Pull request by Nereuxofficial**
Also: Shouldn't "Envionment" be called Environment?

### CODE REVIEW

**Comment by Nereuxofficial** [comment]
Travis CI tried to test the wrong branch...

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ fn main() -> Result<(), Box<dyn Error>> {
         })
         .run(|model, fit, num| {
             println!("Generation: {} score: {:.3?}\t{:?}", num, fit, model.as_string());
-            fit == 12.0 || num == 500
+            (fit == 12.0).abs() < 0.1 || num == 500
```
Comment (martinxyz) [inline README.md:92]: looks like a mistake (`abs()` of a bool)

Comment (Nereuxofficial) [reply README.md:92]: Yup that should be -.

Comment (Nereuxofficial) [reply README.md:92]: You're right, that sould be a - instead of ==

**Pull request merged**